plugin/server/logs.py

```
@@ -0,0 +1,754 @@
+                tail_lines.append(line)
+            
+            # 解析日志行
+            parsed_logs = []
+            for line in tail_lines:
+                log_entry = parse_log_line(line)
+                if log_entry:
+                    parsed_logs.append(log_entry)
+            
+            return parsed_logs
+    except Exception as e:
+        logger.exception(f"Failed to read log file {log_file}: {e}")
```

Ruff (0.14.10)

Redundant exception object included in `logging.exception` call

(TRY401)